packages/docs/src/vite-plugin-synergy-styles/index.ts

```diff
@@ -1,10 +1,15 @@
 import { mkdir, writeFile } from 'fs/promises';
 import { join } from 'path';
+import { execSync } from 'child_process';
 import type { Plugin } from 'vite';
 import type { Config } from './types.js';
 import { getStructure } from './structure.js';
 import { toCem } from './toCem.js';
 
+const runStylesBuild = () => execSync(
```

Comment:
As this is called in async context only, you could also go and use something like

```javascript
const runStylesBuild = async () => exec('...');
```

instead and await its result. Does not come to a different outcome, though :)

Reply:
I just tested it with the `exec` method and for me it endlessly retriggers the `load` function when running it with `pnpm start`. But maybe I have also done something wrong  😅

![Bildschirmfoto 2024-07-01 um 17 15 35](https://github.com/synergy-design-system/synergy-design-system/assets/115139104/c76cd9a6-189c-4d49-9830-67beab0d8862)

Reply:
Uff, then lets just stay with the sync version, its pure semantics overall, so that does not matter much :)